deepchem/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,41 @@
+import pytorch_lightning as pl
```

Comment:
Lightning should be a conditional import. See other torch imports for an example